frontend/build-and-copy.sh

```diff
@@ -2,15 +2,19 @@
 
 set -e
 
-FROM=./result
+RESULT=./result
+OUT=./out
 TO=./../api/src/treestatus_api/static/ui
 
-rm -f $FROM
-nix-build -o $FROM
+rm -f $RESULT
+rm -rf $OUT
+mkdir $OUT
+docker container rm treestatus || :
+docker run --name treestatus -v $(pwd):/src nixos/nix sh -c "cd /src && nix-build -o $RESULT"
+# pipe through tar to avoid permission denied issues
+docker container cp treestatus:$(readlink -f $RESULT)/ - | tar -C $OUT --strip-components=1 -x
```

Comment:
Looks like this is not portable to Mac:
```
readlink: illegal option -- f
usage: readlink [-n] [file ...]
```
Removing the -f works ok though, eg /nix/store/cwc77nz6nagcm0ibbwf72kiz4yprxva4-mozilla-releng-frontend-1.0.0. So perhaps an if block based on `uname -s`.

I do have gnu readlink (via brew's coreutils as greadlink) but it doesn't like the symlink because /nix doesn't exist.

Reply:
Probably I can remove `chown`, because there is no copy anymore, piping through `tar` fixes all the things :)